### ISSUE TITLE
.tns-lazy-img leaving images at .6 opacity

### ISSUE THREAD

**Issue by dolbex** [ganlanyuan/tiny-slider#346]
.tns-lazy-img leaving images at .6 opacity. The image is being loaded but never brought into full opacity

Settings:
```
    this.slider = tns({
      container: this.container,
      items: 1,
      slideBy: "page",
      edgePadding: 0,
      gutter: 0,
      lazyload: true,
      autoplay: false,
      controlsContainer: this.controlsContainer
    });
```

Version: 2.8.8
Chrome 70
MacOS Mojave 10.14

**Comment by ganlanyuan** [comment]
I couldn't reproduce this issue, do you think you can make a demo?

**Comment by lotyrin** [comment]
I am experiencing this, in Firefox only. The image receives 'failed' class instead of 'loaded' (despite the image succeeding to load) which leaves it partially transparent. I will see if I can produce a minimal reproduction case.

**Comment by lotyrin** [comment]
https://codepen.io/lotyrin/full/zyxzyG Demonstrates the issue

**Comment by ganlanyuan** [comment]
Fixed in 45c4bd8

**Comment by dolbex** [comment]
Many thanks @ganlanyuan !!!
